src/Core/Domain/Manufacturer/Command/DeleteManufacturerLogoImageCommand.php

```diff
@@ -0,0 +1,61 @@
+<?php
```

Comment:
As the code of this class and the code of `DeleteManufacturerLogoImageHandler` is the same, is that possible / useful to refactor them?

Reply:
I don't know if it exists Abstract through domain in the project. (I hope you understand me).

Reply:
I check. But I have not found anything. An idea @jolelievre ?